client/client_test.go

```diff
@@ -11,13 +11,15 @@ import (
 	"time"
 
 	onet "github.com/Jigsaw-Code/outline-ss-server/net"
+	ss "github.com/Jigsaw-Code/outline-ss-server/shadowsocks"
 	"github.com/shadowsocks/go-shadowsocks2/shadowaead"
 	"github.com/shadowsocks/go-shadowsocks2/socks"
 )
 
 const (
 	testPassword   = "testPassword"
 	testTargetAddr = "test.local:1111"
+	testCipher     = "chacha20-ietf-poly1305"
```

Comment:
This is probably fine, but it's odd that `testCipher` is redefined here, even though it's the same as `ss.TestCipher`, but `integration_test` just reuses `ss.TestCipher`.  Is there a reason why these tests should redefine `testCipher`, but `integration_test` should not?

Reply:
Changed to `ss.TestCipher`